### PR TITLE
Update to v6.5.3:

### PR DESCRIPTION
**Pull request by akabanovs**
- version bumped up and sha256 updated
 - explicit pinning in host section
 - removed nbclassic from test requirements

### CODE REVIEW

**Comment by akabanovs** [comment]
nbclassic seems to be unused. It is also not mentioned in the upstream in the test requirements, although conda-forge still lists it. I have removed it.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -35,7 +34,7 @@ requirements:
     - jinja2
     - jupyter_client >=5.3.4
     - jupyter_core >=4.6.1
-    - nbclassic >={{ min_nbclassic }}
+    - nbclassic >=0.4.7
     - nbconvert >=5
     - nbformat
     - nest-asyncio >=1.5
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml:40]: upstream has depenedecies on:
"jupyter_server>=2.0.1,<3",
    "importlib-resources>=5.0;python_version<\"3.9\"",
    "jupyterlab>=4.0.0a37,<5",
    "jupyterlab_server>=2.19.0,<3",
    "notebook_shim>=0.2,<0.3",
    "tornado>=6.2.0"
see https://github.com/jupyter/notebook/blob/main/pyproject.toml ,or I miss something here ?

**Review by M-Waszkiewicz-Anaconda — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (akabanovs) [reply recipe/meta.yaml:40]: @M-Waszkiewicz-Anaconda you are looking at the main branch which is already probably ahead of the v6.5.3 release. Here is the toml file for the v6.5.3 we are updating to:
https://github.com/jupyter/notebook/blob/v6.5.3/pyproject.toml

And runtime dependencies are specified in the setup.py file:
https://github.com/jupyter/notebook/blob/v6.5.3/setup.py#L107

Comment (M-Waszkiewicz-Anaconda) [reply recipe/meta.yaml:40]: ahh, OK, that explains

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Pull request merged**